src/app/component/quiz/quiz.component.ts

```diff
@@ -219,7 +222,7 @@ export class Quizcomponent implements OnInit, OnDestroy {
         this.points = this.points -= 0.25;
         this.inCorrectanswer++;
       }
-      if (questionindex === this.question.length) {
+      if (questionindex + 1 === this.question.length) {
```

Comment:
I dont think so cehcking this condition twice would be fine as checking on either answer of check on next

Reply:
This condition is already removed in branch number TATB - 35